### DISCUSSION TITLE
批量更新点和边

### DISCUSSION THREAD

**Discussion by Peng-89** [TuGraph-family/tugraph-db#182] [Q&A]
现在3.4版本是否支持批量更新点和边，如果不支持？后续是否有计划支持

**Comment by knightast** [comment]
https://github.com/TuGraph-family/tugraph-db/issues/179

也可以用graph_import工具，或手写存储过程

**Reply by shenwenquan2023** [reply]
现在有bolt高可用集群，点边的csv文件及相关配置，如何在不登录服务器的情况下，用api的方法将数据快速导入到集群？
